pkgs/development/python-modules/spacy/default.nix

```diff
@@ -56,16 +56,16 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.cfg \
-      --replace "blis>=0.4.0,<0.5.0" "blis>=0.4.0,<1.0" \
+      --replace "blis>=0.4.0,<0.8.0" "blis>=0.4.0,<1.0" \
       --replace "catalogue>=0.0.7,<1.1.0" "catalogue>=0.0.7,<3.0" \
       --replace "plac>=0.9.6,<1.2.0" "plac>=0.9.6,<2.0" \
       --replace "srsly>=1.0.2,<1.1.0" "srsly>=1.0.2,<3.0" \
-      --replace "thinc==7.4.1" "thinc>=7.4.1,<8"
+      --replace "thinc>=7.4.1,<7.5.0" "thinc>=7.4.1,<8"
   '';
 
   pythonImportsCheck = [ "spacy" ];
 
-  passthru.tests = callPackage ./annotation-test {};
+  passthru.tests.annotation = callPackage ./annotation-test { };
```

Comment:
I don't know what this does, honestly. A comment might be helpful?

Reply:
About what part? `passtru.tests` is documented in the nixpkgs manual. `annotation` is the name of the test, because it is an annotation test.

Reply:
more accurately, nix build commands can also build a set of derivations. This is just one way to organize many tests.

Reply:
👍 I'd usually not seen it in this format, so I got confused